obspy_github_api/obspy_github_api.py

```diff
@@ -67,18 +80,29 @@ def check_specific_module_tests_requested(issue_number):
     return modules_to_test
 
 
-def get_module_test_list(issue_number):
+def get_module_test_list(
+    issue_number, token=None, module_path="./obspy/core/util/base.py"
+):
```

Comment:
that's a very sad looking line of code right there

Reply:
Ya, it does look worse than before. But, it does something very important; it allows us to extract a module list *without* having obspy installed. This wasn't possible before.

Reply:
i was just jokin about this line of code..

```
):
```
But in general I think we have the closing stuff on the same line.. my feeling tells me this is `black` styling?

Reply:
Oh, I thought you meant about expanding the number of arguments. Yes, this is a black styling thing, I guess I had just gotten used to it.